nginx/templates/vhost-base.conf

```diff
@@ -7,8 +7,16 @@
 
 {% block server %}
 server {
+{% if nginx.secure %}
+    listen     443 ssl;
+    server_name  {{server_name|default('_')}};
+    ssl on;
+    ssl_certificate      /etc/pki/{{grains['opg_project']}}/{{grains['opg_project']}}.crt;
+    ssl_certificate_key  /etc/pki/{{grains['opg_project']}}/{{grains['opg_project']}}.key;
+{% else %}
```

Comment:
Two things here:

1) `opg_project` is the wrong grain name here  to use so this name would need changing - it's OPG specific and this formula needs to be generic; it's not clear that the grain would have this affect. But:
2) We already do SSL for nginx (and keep a better list of ciphers and settings to make it more secure than the defaults.) in https://github.com/ministryofjustice/sslloadbalancer-formula - if you want nginx to terminate SSL then you should use that formula (which is just a layer on top of this one)